vellum/workflows/triage_message/nodes/update_contact_summary_node.py

```diff
@@ -0,0 +1,59 @@
+from vellum import (
+    ChatMessagePromptBlock,
+    JinjaPromptBlock,
+    PromptParameters,
+)
+from vellum.workflows.nodes import InlinePromptNode
+from .read_message_node import ReadMessageNode
+from .fetch_contact_summary_node import FetchContactSummaryNode
+
+
+class UpdateContactSummaryNode(InlinePromptNode):
+    ml_model = "gpt-4o"
+    blocks = [
+        ChatMessagePromptBlock(
+            chat_role="SYSTEM",
+            blocks=[
+                JinjaPromptBlock(
+                    template="""You are updating a contact summary based on a new message from the contact. \
+Your goal is to maintain a concise, informative summary of this contact that captures:
+- Their relationship to Vargas JR (client, prospect, recruiter, etc.)
+- Key topics they've discussed or requested
+- Any important context about their needs or interests
+- Notable patterns in their communication
+
+Keep the summary focused and actionable. Update or add new information based on the latest message, \
+but preserve important historical context. The summary should be 2-4 sentences.
+
+{% if current_summary %}Current summary:
+{{ current_summary }}
+{% else %}This is the first message from this contact, so there is no existing summary.
+{% endif %}
+
+Contact details:
+{% if contact_full_name %}- Full name: {{ contact_full_name }}
+{% endif %}{% if contact_email %}- Email: {{ contact_email }}
+{% endif %}{% if contact_slack_display_name %}- Slack display name: {{ contact_slack_display_name }}
+{% endif %}{% if contact_phone_number %}- Phone number: {{ contact_phone_number }}
+{% endif %}
+
+Latest message (via {{ channel }}):
+{{ message }}
+
+Provide an updated summary that incorporates this new information.""",
```

Comment:
Add some guidance here at the end saying that it's likely okay if there are no necessary updates needed if the new message is pretty aligned with the current summary